src/registrar/assets/js/get-gov.js

```diff
@@ -1873,6 +1873,88 @@ class MembersTable extends LoadTableBase {
   }
+  
+  initShowMoreButtons() {
+    function toggleShowMoreButton(toggleButton, contentDiv, buttonParentRow) {
+      const spanElement = toggleButton.querySelector('span');
+      const useElement = toggleButton.querySelector('use');
+      if (contentDiv.classList.contains('display-none')) {
+        showElement(contentDiv);
+        spanElement.textContent = 'Close';
+        useElement.setAttribute('xlink:href', '/public/img/sprite.svg#expand_less');
+        buttonParentRow.classList.add('hide-td-borders');
+        toggleButton.setAttribute('aria-label', 'Close additional information');
+      } else {    
+        hideElement(contentDiv);
+        spanElement.textContent = 'Expand';
+        useElement.setAttribute('xlink:href', '/public/img/sprite.svg#expand_more');
+        buttonParentRow.classList.remove('hide-td-borders');
+        toggleButton.setAttribute('aria-label', 'Expand for additional information');
+      }
+    }
+  
+    let toggleButtons = document.querySelectorAll('.usa-button--show-more-button');
+    toggleButtons.forEach((toggleButton) => {
+      
+      let dataFor = toggleButton.dataset.for;
+      let contentDiv = document.getElementById(dataFor);
+      let buttonParentRow = toggleButton.parentElement.parentElement;
+      if (contentDiv && contentDiv.tagName.toLowerCase() === 'tr' && contentDiv.classList.contains('show-more-content') && buttonParentRow && buttonParentRow.tagName.toLowerCase() === 'tr') {
+        toggleButton.addEventListener('click', function() {
+          toggleShowMoreButton(toggleButton, contentDiv, buttonParentRow);
+        });
+      } else {
+        console.warn('Found a toggle button with no associated toggleable content or parent row');
+      }
+
+    });
+  }
+
+  /**
+   * Helper function which takes last_active and returns display value and sort value
+   * @param {*} last_active - UTC date, or strings "Invited" or "Invalid date"
+   * @returns 
+   */
+  handleLastActive(last_active) {
+    let last_active_display = '';
+    let last_active_sort_value = -1; // sort by default as numeric value to sort with dates
+  
+    const invited = 'Invited';
+    const invalid_date = 'Invalid date';
+    const options = { year: 'numeric', month: 'long', day: 'numeric' }; // Format options for date
+  
+    // Check if last_active is not null
+    if (last_active) {
+      if (last_active === invited) {
+        last_active_display = invited;
+        last_active_sort_value = 0; // sort as numeric value 
+      } else if (last_active === invalid_date) {
+        last_active_display = invalid_date;
+      } else {
+        const parsedDate = new Date(last_active);
+        
+        try {
+          if (!isNaN(parsedDate.getTime())) { // Check if the date is valid
+            last_active_display = parsedDate.toLocaleDateString('en-US', options);
+            last_active_sort_value = parsedDate.getTime(); // sort as numeric value, seconds since 1970
```

Comment:
We have to sort dates as well as string values, namely 'Invalid date' and 'Invited'. Assigned each of these a numeric value (0 and 1) and sorted dates according to time since epoch